helm-charts/charts/kube-starrocks/templates/init-pwd/job.yaml

```diff
@@ -0,0 +1,50 @@
+{{- if .Values.initPassword.enabled }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "kube-starrocks.name" . }}-credential
+  namespace: {{ .Release.Namespace }}
+stringData:
+  password: {{ .Values.initPassword.password }}
```

Comment:
use b64enc to encode ?  If not, ` helm template -f values.yaml . | kubectl apply -f -`  will faile. 
And If you do encode, the environment will be the encoded format.

Reply:
Is that what you mean?
```
data:
  password: {{ .Values.initPassword.password | b64enc }}
```

Reply:
yes

Reply:
MYSQL_PWD environment variable is used by MySQL and cannot pass in encoded strings. If you want to solve the problem of starting after changing the root password in a different way, it may be a different issue. So according to my understanding, the above approach may be more feasible.

Reply:
done